sapp/ui/frontend/package.json

```diff
@@ -79,7 +79,12 @@
     "flow": "flow"
   },
   "eslintConfig": {
-    "extends": "react-app"
+    "extends": "react-app",
+    "rules": {
+      "eqeqeq": "off",
```

Comment:
Why we disabled these?

Reply:
Javascript has two operators ==, === one compares values and the other compares value and type. Both are useful. JSLint just prefers the use of === over == and complains when we use ==. Since changing all == to === can break things, I thought it is nice to avoid the check.

https://stackoverflow.com/questions/3735939/jslint-expected-and-instead-saw

But yeah, if we want to just use === throughout, I am okay to remove it :)